es5-shim.js

```diff
@@ -328,6 +328,54 @@ var strSplit = call.bind(StringPrototype.split);
 var strIndexOf = call.bind(StringPrototype.indexOf);
 var push = call.bind(array_push);
 
+// ES-5 15.3.4.4
+// http://es5.github.io/#x15.3.4.4
+var hasToStringTagBasicBug = toStr() !== '[object Undefined]' || toStr(null) !== '[object Null]';
+var hasToStringTagLegacyArguments = toStr(arguments) !== '[object Arguments]';
+var hasToStringTagInconsistency = hasToStringTagBasicBug || hasToStringTagLegacyArguments;
+
+// Others to check, or could be fixed:
+// PhantomJS 1.9 which returns 'function' for `NodeList` instances.
+// Older ES3 native functions like `alert` return `[object Object]`.
+
+if (hasToStringTagLegacyArguments) {
+  // This function is for use within `Function#call` only.
+  var isDuckTypeArguments = function (value) {
+    return value != null && // Checks `null` or `undefined`.
+        typeof value === 'object' &&
+        owns(value, 'length') &&
+        typeof value.length === 'number' &&
+        value.length >= 0 &&
+        !owns(value, 'arguments') &&
+        owns(value, 'callee');
+        // Using `isCallable` here is dangerous as it can cause recursion.
+        // A typeof check would be ok if necessary.
+  };
+}
+
+defineProperties(FunctionPrototype, {
+  call: function () {
+      var thisArg = arguments[0];
```

Comment:
`Function.call` has a length of `1`, so `thisArg` should be a named argument. http://www.ecma-international.org/ecma-262/5.1/#sec-15.3.4.4